src/stories/Library/review/review.scss

```diff
@@ -0,0 +1,36 @@
+.reviews {
+  margin: 0 $s-4xl 0 $s-4xl;
+}
+
+.review {
+  margin-bottom: $s-xl;
+  &:last-of-type {
+    margin-bottom: $s-2xl;
+  }
+
+  &__meta {
+    color: $c-text-secondary-gray;
+  }
+
+  &__heart {
+    // in Figma the heart is 12px but the img SVG in the browser has
+    // some white space in the top, so I went for making it visually
+    // the same height as the font to match the design brief
+    height: 14px;
+    display: inline-block;
+    margin-right: 2px;
+    &:last-of-type {
+      margin-right: 0;
+    }
+  }
+
+  &__headline {
+    color: $c-text-secondary-gray;
+  }
+
+  &__body {
+    max-width: 545px;
+    line-height: 16px;
```

Comment:
Consider if we should add `.text-body-small-medium` or `.text-body-small-regular` to the element instead of specifying the line height for the review body text.

It seems only specific that we should add this here instead of using the general typography setup.

Reply:
Mmm well the general typography is not consistent with the design brief. The block of text is already inheriting  .text-small-caption as specified that it should. All fits but the line height, so I adjusted it for the specific block of text. It is defined a bit weirdly in the typography classes as 120%, which just doesn't match. And upon testing it out with and without adjustments - it is easier to read for sure when we follow the design brief.
And I didn't want to go down the road of adjusting the predefined class 😅 

But good spotted and great point!